dc/s2n-quic-dc/Cargo.toml

```diff
@@ -20,7 +20,7 @@ arrayvec = "0.7"
 atomic-waker = "1"
 aws-lc-rs = "1"
 bitflags = "2"
-bolero-generator = { workspace = true, optional = true }
+bolero-generator = { version = "0.12", optional = true }
```

Comment:
I think since `default-features = false` was specified in the `workspace.dependencies`, to replicate the same behavior you would say `default-features = false` everywhere that `bolero-generator` is used